src/CommandingOfficers/Meridian.java

```diff
@@ -148,6 +151,20 @@ protected void perform(MapMaster gameMap)
         unit.isTurnOver = false;
       }
     }
+
+    @Override
+    public Collection<DamagePopup> getDamagePopups(GameMap gameMap)
+    {
+      ArrayList<DamagePopup> output = new ArrayList<DamagePopup>();
+
+      for( Unit unit : COcast.justTransformed )
+        output.add(new DamagePopup(
+                       new XYCoord(unit.x, unit.y),
+                       myCommander.myColor,
+                       "Flow"));
```

Comment:
Don't think popups are really needed for this one since the application rules are simple.
Plus "Flow" may not be the most descriptive thing anyway. Maybe "Act", if we can't go over 4 chars? I dunno, I'd just remove it.

Reply:
There's no other indicator, though, and I don't want to make the user have to remember (especially as you can save in the middle of a turn).
We can go over 4 characters, I just thought "Flow" went well since it's in the power name :V